Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for a fresh installation on Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide.
+For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Get rkt repo and install dependencies
+In this example ~/Repos is a personal workspace where all repos are stored
+```sh
+$ mkdir ~/Repos && cd ~/Repos
+$ mkdir -p ~/.local/gopath/src/github.com/coreos
+$ sudo apt-get install git
+$ git -C ~/.local/gopath/src/github.com/coreos clone https://github.com/coreos/rkt.git
+$ ln -s ~/.local/gopath/src/github.com/coreos/rkt rkt
+```
+On a fresh system installation, few additional software packages are needed to correctly build rkt:
+```sh
+$ sudo ~/Repos/rkt/scripts/install-deps-debian-sid.sh
+```
+See also the dependencies [page](https://github.com/coreos/rkt/blob/master/Documentation/dependencies.md).
+### Installing Go Programming Language for a single-user
+```
+$ cd ~/Downloads
+$ wget https://storage.googleapis.com/golang/go1.6.1.linux-amd64.tar.gz
+$ tar -xvf go1.6.1.linux-amd64.tar.gz
+$ mv go ~/.local
+```
+
+Add GO variables to .bashrc file:
+```sh
+export PATH=~/.local/bin:~/.local/go/bin:$PATH
+export GOPATH=~/.local/gopath
+export GOROOT=~/.local/go
+```
+
+### Install ccache (optional step)
+Ccache can save a lot of time. If you build a kernel once, most of the compiled code can just be taken from the cache.
+Ccache can be configured in a few easy steps:
+
+```sh
+$ sudo apt-get install ccache
+$ ccache --max-size=10G
+$ sudo ln -s /usr/bin/ccache /usr/local/bin/gcc
+```
+The maximum cache size is 10GB now (the default value is too small to cache kernel compilation).
+
+### Building rkt
+
+Run the autogen and configure commands with the relevant arguments, for example (kvm as flavor):
+```sh
+$ cd ~/Repos/rkt
+$ ./autogen.sh && ./configure --enable-functional-tests --with-stage1-flavors=kvm
+```
+Now build rkt with:
+```sh
+$ make V=2 -j
+```
+REMEMBER: If you want to test somebody else's changes:
+```sh
+$ git checkout <branch>
+$ make clean
+$ ./autogen.sh && ./configure <proper arguments>
+```
+### A few useful commands
```

Comment:
Please reformat this section as a level two heading with several level three headings below it, e.g,:

## Useful commands

### Build and run only tests

```sh
$ ./tests/build-and-run-tests.sh -f kvm
```

### Run functional tests only after a build

...
```